test/scopes/ProxyUser.t.sol

```diff
@@ -274,12 +274,26 @@ abstract contract ProxyUser is BaseUser, Contracts, ScriptBase {
     uint256 _coinsToExit = safeEngine.coinBalance(address(_proxy));
 
     bytes memory _callData =
-      abi.encodeWithSelector(CommonActions.exitSystemCoins.selector, address(coinJoin), _coinsToExit);
+    // REVIEW: collectSystemCoins vs. exitSystemCoins
```

Comment:
collect is: send to proxy + exit
exit is: call coinJoin.exit (coins should already be on the proxy)

Reply:
`collectTokenCollateral()`, which transfers collateral from the SAFE handler to the proxy and then to the user, now exists within the `BasicActions`.

Nonetheless, I noticed that `collectProtocolTokens()`, which transfers protocol tokens from the proxy to the user, already existed within the `DebtBidActions`.

Thus, taking into account the provided descriptions of `collect` and `exit`, I find a naming convention contradiction between the former and the latter.

Reply:
hmmm because there's no Join for the protocol token i guess, but yes, i agree that it may be an edge case to the convention